README.md

```diff
@@ -18,6 +18,7 @@ the details of all the apps involved check
 - [How to run the tests](#how-to-run-the-tests)
 - [Dealing with a flaky test](#dealing-with-a-flaky-test)
 - [Contributing](#contributing)
+- [Help with debugging a Jenkins failure](./docs/debugging-failures.md)
```

Comment:
I'm conscious that this isn't really a "Contents" item. Will think weaving this link in some other way.

Reply:
Sure. We could have a section under contents of useful docs or something and could bung in:

docker guide
contributing
this

and any others that grab you?